apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -4,13 +4,25 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
 
   @callables [Result.Function, Result.Macro]
 
+  def completion(%callable_module{argument_names: []} = callable, %Env{} = env)
```

Comment:
why are you looking at the argument names, when you have an `arity` key in callables? That's a lot clearer, imo:
```elixir
def completion(%callable_module{arity: 0}, %Env{} = env) do 
...
```